test/src/test/scala/shapeless/datatype/test/Records.scala

```diff
@@ -5,22 +5,26 @@ import org.joda.time.Instant
 import org.scalacheck._
 
 object Records {
+  case class UserId(id: Long)
+
+  case class User(userId: UserId, name: String)
+
   case class Required(booleanField: Boolean,
                       intField: Int, longField: Long, floatField: Float, doubleField: Double,
                       stringField: String,
-                      byteStringField: ByteString, byteArrayField: Array[Byte],
```

Comment:
Why removing these `byteStringField: List[ByteString]` here? Are you trying to decouple protobuf dependency?